alibi/tests/utils.py

```diff
@@ -12,8 +12,10 @@ class MockPredictor:
     allow testing of functionality that depends on it without
     inference overhead.
     """
-    def __init__(self, out_dim: int,
+    def __init__(self,
+                 out_dim: int,
                  out_type: str = 'proba',
+                 model_type: str = None,
```

Comment:
What is `model_type` for?

Side note, I've extended the functionality of `MockPredictor` in the ALE PR to handle regression.

Reply:
TreeExplainer sets a `model_type` property that indicates what library a given model is for (e.g., `catboost`)